docs/digger/admin/managing-digger-logs-artifacts.asciidoc

```diff
@@ -0,0 +1,31 @@
+== Managing Digger Build Logs and Artifacts
+App builds started by Digger produce build logs and build artifacts.
+Over time, these will build up. There are a number of options to manage the buildup. If you are using Digger with RHMAP, these options can be specified
+by setting environment variables in the `millicore` deployment(???) configuration in
```

Comment:
this relates to an instance of millicore running in an rhmap-core OpenShift instance. These environment variables are set by the ansible playbook [here](https://github.com/fheng/rhmap-ansible/blob/master/playbooks/buildfarm.yml#L97-L138) We aren't currently supporting it in digger-installer(opensource version) as the core of rhmap is not currently opensource.